webrtc-stats.html

```diff
@@ -1202,6 +1202,8 @@
              boolean             ended;
              boolean             detached;
              DOMString           kind;
+             DOMHighResTimeStamp playoutTimestamp;
+             DOMHighResTimeStamp captureTimestamp;
```

Comment:
I still prefer these to be estimatedPlayoutTimestamp and estimatedCaptureTimestamp. I know we won't get a better estimate, but with the names as it it sounds like it is the last frame rather then what we estimate is the current one.